File: app/src/main/java/dev/ragnarok/fenrir/dialog/base/BaseDialogFragment.java
package dev.ragnarok.fenrir.dialog.base;

import androidx.fragment.app.DialogFragment;

import dev.ragnarok.fenrir.util.ViewUtils;

public abstract class BaseDialogFragment extends DialogFragment {

    @Override
    public void onDestroy() {
        super.onDestroy();
        ViewUtils.keyboardHide(requireActivity());
    }
}

File: app/src/main/java/dev/ragnarok/fenrir/dialog/base/BaseDialogFragment.kt
package dev.ragnarok.fenrir.dialog.base

import androidx.fragment.app.DialogFragment
import dev.ragnarok.fenrir.util.ViewUtils.keyboardHide

abstract class BaseDialogFragment : DialogFragment() {
    override fun onDestroy() {
        super.onDestroy()
        keyboardHide(requireActivity())
    }
}